test/services/database/comment_repository_test.dart

```diff
@@ -1,298 +0,0 @@
-          id: commentId,
-          data: commentData,
-        );
-
-        // Check that the comment was added correctly
-        final actualComments = await commentRepository.getComments(postId);
-
-        expect(
-          actualComments,
-          unorderedEquals(alreadyPresentComments + [expectedNewComment]),
-        );
-
-        // Check that the comment count was updated correctly
-        final postDoc = await postDocument.get();
-        final post = PostFirestore.fromDb(postDoc);
-
-        expect(post.data.commentCount, equals(alreadyPresentCommentsCount + 1));
-      });
-    });
-
-    group("deleting comments", () {
-      test("should delete a comment from a post with single comment", () async {
-        final comment =
-            await commentGenerator.addComments(1, postId, commentRepository);
-
-        final commentId = comment.first.id;
-
-        // Add an upvote to the comment to later check that it is deleted
-        commentUpvoteRepository.setUpvoteState(
-          testingUserFirestoreId,
-          commentId,
-          VoteState.upvoted,
-        );
-
-        await commentRepository.deleteComment(postId, commentId);
-
-        // Check that the comment was deleted
-        final actualComments = await commentRepository.getComments(postId);
-        expect(actualComments, isEmpty);
-
-        // Check that the upvote was deleted
-        final upvote = await commentUpvoteRepository.getUpvoteState(
-          testingUserFirestoreId,
-          commentId,
-        );
-        expect(upvote, VoteState.none);
-
-        // Check that the comment count was updated correctly
-        final postDoc = await postDocument.get();
-        final post = PostFirestore.fromDb(postDoc);
-
-        expect(post.data.commentCount, equals(0));
-      });
-
-      test("should delete a comment from a post with multiple comments",
-          () async {
-        const alreadyPresentCommentsCount = 5;
-
-        final comments = await commentGenerator.addComments(
-          alreadyPresentCommentsCount,
-          postId,
-          commentRepository,
-        );
-
-        // Delete the third comment
-        final commentToDeleteId = comments[2].id;
-        await commentRepository.deleteComment(postId, commentToDeleteId);
-
-        final expectedComments =
-            comments.where((c) => c.id != commentToDeleteId);
-
-        // Check that the comment was deleted
-        final actualComments = await commentRepository.getComments(postId);
-
-        expect(
-          actualComments,
-          unorderedEquals(expectedComments),
-        );
-
-        // Check that the comment count was updated correctly
-        final postDoc = await postDocument.get();
-        final post = PostFirestore.fromDb(postDoc);
-
-        expect(post.data.commentCount, equals(alreadyPresentCommentsCount - 1));
-      });
-
-      test(
-          "should thrown an error and do nothing if the comment does not exist",
```

Comment:
This test is missing in the new file. Is it expected?

Reply:
Fixed in ba41cb876b1e7ac78e33e5302cdf61a525f319d3.

Reply:
Good initiative !
Your implementation looks good to me.
Thanks :+1: